pandas/core/categorical.py

```diff
@@ -199,18 +199,21 @@ class Categorical(PandasObject):
     >>> from pandas import Categorical
     >>> Categorical([1, 2, 3, 1, 2, 3])
     [1, 2, 3, 1, 2, 3]
-    Categories (3, int64): [1 < 2 < 3]
+    Categories (3, int64): [1, 2, 3]
 
     >>> Categorical(['a', 'b', 'c', 'a', 'b', 'c'])
     [a, b, c, a, b, c]
-    Categories (3, object): [a < b < c]
+    Categories (3, object): [a, b, c]
 
     Only ordered `Categoricals` can be sorted (according to the order
```

Comment:
This is actually incorrect: unordered categoricals *can* be sorted, but it is true they do not have a min and max

Reply:
I've changed the wording of the paragraph.